dates.py

```diff
@@ -0,0 +1,29 @@
+from datetime import date
+from log import logger
+
+
+def day_difference():
+    fd, sd = (input('Введите дату в формате "year.month.day"\nПервая дата: '),
+              input('Вторая дата: '))
+    try:
+        year, month, day = map(int, fd.split('.'))
+        n_fd = date(year, month, day)
+        year, month, day = map(int, sd.split('.'))
+        n_sd = date(year, month, day)
+        return abs((n_fd - n_sd).days)
+    except ValueError as e:
+        logger.warning('Неправильно указана дата')
+        return e
+
+
+def past_or_future_date():
```

Comment:
more or less the same: function takes 1 date and returns string
```suggestion
def past_or_future_date(yd):
    ...
```